graphics/libdmtx/Portfile

```diff
@@ -20,9 +20,11 @@ long_description    libdmtx is open source software for reading and writing \
                     Acuity CiMatrix. Wikipedia has a good article on the \
                     symbology and its characteristics.
 
-homepage            http://libdmtx.sourceforge.net
-master_sites        sourceforge:project/libdmtx/libdmtx/${version}/
-use_bzip2           yes
+use_autoreconf      yes
+distfiles           v${version}${extract.suffix}
+homepage            https://github.com/dmtx/libdmtx
+master_sites        ${homepage}/archive/
```

Comment:
Please use the GitHub portgroup which will condense all 3 of the above lines and the `version` into a single setup command. See the existing `JetBrains` Portfile, among many, for an example.

Reply:
Will do and thank you for your feedback! These are my first port files and I need the guidance.